src/mscorlib/model.xml

```diff
@@ -7425,16 +7425,29 @@
       <Member MemberType="Property" Name="TotalMinutes" />
       <Member MemberType="Property" Name="TotalSeconds" />
     </Type>
+    <Type Name="System.TimeZone">
```

Comment:
Apparently, there is no `public abstract class TimeZone` in CoreCLR as defined by full framework.